src/utils/helpers/buildMapper.ts

```diff
@@ -10,17 +10,19 @@ const STATUS_OPTIONS: any = {
 
 const TIMEZONE = Intl.DateTimeFormat().resolvedOptions().timeZone;
 
-const isBuilding = (status: string) => {
-  const BUILD_STATUS = ["BUILDING"];
-  return BUILD_STATUS.includes(status);
-};
-
-const isQueued = (status: string) => {
-  const BUILD_STATUS = ["QUEUED"];
-  return BUILD_STATUS.includes(status);
-};
+const isCompleted = ({
+  status,
+  ended_on,
+  scheduled_on
+}: Pick<Build, "status" | "ended_on" | "scheduled_on">) => {
+  let duration = 0;
+  if (ended_on && scheduled_on) {
+    const startTime = new Date(scheduled_on);
+    const endTime = new Date(ended_on);
+    duration = (endTime.valueOf() - startTime.valueOf()) / 60000;
+    duration = Math.round(duration);
+  }
```

Comment:
I guess you really only need to do this if `status === "COMPLETED"`, so this can be moved inside the branch below.